src/main/java/picard/analysis/CollectMultipleMetrics.java

```diff
@@ -157,13 +156,28 @@ default boolean supportsMetricAccumulationLevel() {
         }
     }
 
-    public enum Program implements ProgramInterface {
+    public enum Program implements ProgramInterface, CommandLineParser.ClpEnum {
+
         CollectAlignmentSummaryMetrics {
+
             @Override
             public boolean supportsMetricAccumulationLevel() {
                 return true;
             }
 
+            final String METRICS_EXTENSION = ".alignment_summary_metrics";
+            final String PDF_EXTENSION = ".read_length_histogram.pdf";
+
+            final List<String> OUTPUT_EXTENSIONS = CollectionUtil.makeList(
+                    METRICS_EXTENSION,
+                    PDF_EXTENSION);
+
+            @Override
+            public String getHelpDoc() {
+                return picard.analysis.CollectInsertSizeMetrics.USAGE_SUMMARY +
```

Comment:
I think this should be `CollectAlignmentSummaryMetrics.USAGE_SUMMARY`

Reply:
oops. good catch.